cli/script/command-executor.ts

```diff
@@ -1288,6 +1295,19 @@ function throwForInvalidOutputFormat(format: string): void {
 function whoami(command: cli.ICommand): Promise<void> {
     return sdk.getAccountInfo()
         .then((account): void => {
-            log(`${account.email} (${account.linkedProviders.join(", ")})`);
+            log(`Account: ${account.email} (${account.linkedProviders.join(", ")})`);
+
```

Comment:
Minor nit, but I'd prefer that we only show the "Account: " prefix if we're also showing the proxy. That way, for the users not using the proxy support, the existing behavior remains the same. But if we do show the proxy, then we label both peices of data.